apps/auction2/templates/index.jade

```diff
@@ -32,8 +32,9 @@ block body
               //- rendered on the client
             .cf-total-sort__sort
               //- rendered on the client
-          #cf-artworks
-            //- rendered on the client
-          hr
-          .cf-pagination
-            //- rendered on the client
+          .cf-bottom-right-part
+            #cf-artworks
+              //- rendered on the client
```

Comment:
IMO it's helpful to know that these aren't just orphaned divs, and that the data/display will be populated client-side.

i.e. in this case: https://github.com/artsy/force/blob/master/components/artwork_brick_rail/templates/index.jade#L13 it's easy to differentiate between which components are `include`-ed vs. those that are rendered client-side.